src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -15,6 +16,7 @@ import TopicDomainHeader from './TopicDomainHeader';
 import ButtonFollowing from './ButtonFollowing';
 import useChatClientHook from '../../../utils/getstream/useChatClientHook';
 import {colors} from '../../../utils/colors';
+import Search from '../../DiscoveryScreenV2/elements/Search';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Search` component seems fine. Make sure that this component is used properly in the code.



<!-- This is an auto-generated comment by OSS CodeRabbit -->